src/test/resources/docker/4.0/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM datastax/cassandra:4.0
+FROM cassandra:4.0
```

Comment:
Changed to the official DockerHub images as the latest in the datastax repo is a pre-4.0 release beta version.